Conta_Bancaria/src/Conta.java

```diff
@@ -75,11 +85,16 @@ public void depositar(double valor) {
 
     // Método para sacar dinheiro da conta
     public boolean sacar(double valor) {
-        if (valor > 0 && valor <= saldo) {
+        double saldoDisponivel = saldo + chequeEspecial; // Saldo disponível inclui o cheque especial
+        if (valor > 0 && valor <= saldoDisponivel) {
```

Comment:
(sugestão) No Clean Code recomenda-se que: "Nomeie expressões booleanas com funções ou variáveis descritivas"

Para isso, você pode utilizar de variavéis ou métodos privados dentro da classe